libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -122,7 +123,48 @@ struct CopyTranslate: public yul::ASTCopier
 		}
 
 		string value;
-		if (varDecl->isStateVariable())
+		if (varDecl->isConstant())
+		{
+			VariableDeclaration const* variable = rootConstVariableDeclaration(*varDecl);
+			solAssert(variable, "");
+
+			if (variable->value()->annotation().type->category() == Type::Category::RationalNumber)
+			{
+				u256 intValue = dynamic_cast<RationalNumberType const&>(*variable->value()->annotation().type).literalValue(nullptr);
+				if (FixedBytesType const* bytesType = dynamic_cast<FixedBytesType const*>(variable->type()))
```

Comment:
Just in case you want to follow `clang-tidy` not duplicating type name :)

```suggestion
				if (auto const* bytesType = dynamic_cast<FixedBytesType const*>(variable->type()))
```